src/geo/transform.js

```diff
@@ -16,10 +16,19 @@ import assert from 'assert';
 import {UnwrappedTileID, OverscaledTileID, CanonicalTileID} from '../source/tile_id.js';
 import type {Elevation} from '../terrain/elevation.js';
 import type {PaddingOptions} from './edge_insets.js';
+import type Map from '../ui/map.js';
```

Comment:
Unused?

Reply:
Ah, yes, was a result of the other raycasting function, I think.